### ISSUE TITLE
Add table of drawing operations to docs

### ISSUE THREAD

**Issue by rfuest** [jamwaffles/embedded-graphics#223]
From https://github.com/jamwaffles/embedded-graphics/pull/222#pullrequestreview-335126128:
>Forgot we had DrawTarget::draw_pixel! Looks good to me, but please add an entry to the changelog under Added - it's a public API change that someone out there might want to know about.

The ability to be able to draw single pixels by using `Pixel(...).draw(&mut display)` isn't very discoverable IMO. I've added some examples to the docs, but they are hard to find inside the `drawable` module.

I suggest to add a table to the front page of the docs, which provides a visual reference of the supported drawing operations:

| Image | Example |
|--------|-----------|
| (Image of a single pixel) | <p>Link to Pixel</p> **Example:** <br> `Pixel(...).draw(&mut display)` |
| (Image of a rectangle) | <p>Link to Rectangle</p> **Example:** <br> `Rectangle(...).draw(&mut display)` |
| ... | ... |

**Comment by jamwaffles** [comment]
A table is a really good idea, particularly with screenshots. Visual docs for a visual library 👍.

If they get added to the main page docs, I think we need to do something like the hack in `check_readme.md` and include that huge doc comment from an external markdown file. `lib.rs` is getting pretty long already, and editing lots of text in `///` comments gets painful fast.

**Comment by jamwaffles** [comment]
To add: the screenshots should be of the simulator with window title. In the example above, `(Image of a single pixel)` is going to look very strange without any context.

**Comment by jamwaffles** [comment]
This is the last meaty issue before we can put 0.6.0 out I think. Do we still want to get it done before release? If so I'm happy to take a stab at it. Do you have any suggestions on how to automatically generate the screenshots?

**Comment by rfuest** [comment]
I wouldn't mind if this was postponed. And this might also be a good candidate for the appendix of an eventual future e-g book.

But if you want to take a stab at it go ahead. I would create a program that generates the screenshots using multiple `display.to_image_buffer(&output_settings).save('some.png')` calls (and not the environment variable). This program could be called manually, because the output doesn't change very often and automating this further won't be worth the effort IMO.

**Comment by jamwaffles** [comment]
I made a start in #279 but it's becoming a bit of a can of worms to do it in a maintainable way I think. I'm going to postpone this as well as #279 for after 0.6.0 release, which I'll do later today (:tada:)
